parlai/core/agents.py

```diff
@@ -343,8 +343,21 @@ def create_agent(opt, requireModelExists=False):
         from parlai.core.params import ParlaiParser
         parser = ParlaiParser(add_parlai_args=False)
         parser.add_parlai_data_path()
+        # add model args if they are missing
```

Comment:
thnx! maybe we can move some code out into another function somehow? this is getting awful long!